syft/workers/base.py

```diff
@@ -197,7 +197,7 @@ def load_data(self, data: List[Union[torch.Tensor, AbstractTensor]]) -> None:
                 self.register_obj(tensor)
                 tensor.owner = self
 
-    def send_msg(self, msg_type: int, message: str, location: "BaseWorker") -> object:
+    def send_msg(self, message: "messaging.Message", location: "BaseWorker") -> object:
```

Comment:
what is the circle of imports here?

Reply:
what do you mean?

Reply:
why is this a string for the class?

Reply:
"AttributeError: module 'syft.messaging' has no attribute 'Message'"

Reply:
I think it has to do with the order in which things are processed. Type annotations come super early before a lot of the __init__ has been created.

Reply:
All good